bezier-rs/docs/interactive-docs/src/utils/drawing.ts

```diff
@@ -94,10 +110,11 @@ export const drawBezier = (ctx: CanvasRenderingContext2D, points: Point[], dragI
 	}
 	ctx.stroke();
 
-	drawLine(ctx, start, handleStart, strokeColor);
-	drawLine(ctx, end, handleEnd, strokeColor);
+	drawLine(ctx, start, handleStart, styleConfig.handleLineStrokeColor);
+	drawLine(ctx, end, handleEnd, styleConfig.handleLineStrokeColor);
 
 	points.forEach((point, index) => {
-		drawPoint(ctx, point, getPointSizeByIndex(index, points.length, radius), index === dragIndex ? COLORS.INTERACTIVE.SELECTED : strokeColor);
+		const strokeColor = index === 0 || index === points.length - 1 ? styleConfig.curveStrokeColor : styleConfig.handleStrokeColor;
```

Comment:
Please break out the `index === 0 || index === points.length - 1` into a named variable so the ternary is easier to read and so that boolean condition can be self-documenting with a name.